src/pages/index.tsx

```diff
@@ -44,6 +44,11 @@ export default Login;
 const AuthShowcase: React.FC = () => {
   const { data: sessionData } = useSession();
 
+  const handleSignOut = async () => {
+    localStorage.polisUserXID = undefined;
```

Comment:
```suggestion
    localStorage.clear();
```

Maybe we should just clear all the keys